stable/concourse/templates/web-svc.yaml

```diff
@@ -18,13 +18,13 @@ metadata:
 spec:
   type: {{ .Values.web.service.type }}
   ports:
-    - name: atc
+    - name: {{ .Values.concourse.atcPortName }}
```

Comment:
If you're looking to name the ports in a similar or the same manner to the service would it make sense to base it on the service name without an extra property? Something like what the [sentry chart](https://github.com/kubernetes/charts/blob/cb0060835763bcdd68fb080ab4799f36a1033fa1/stable/sentry/templates/service.yaml#L16) does?

Reply:
Nice I can update the PR to do something like sentry